src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/AwsSdk/AwsSdkPipelineWrapper.cs

```diff
@@ -54,12 +54,25 @@ public AfterWrappedMethodDelegate BeforeWrappedMethod(InstrumentedMethodCall ins
                 return Delegates.NoOp;
             }
             dynamic request = requestContext.OriginalRequest;
-            string requestType = request.GetType().Name;
+            string requestType = request.GetType().FullName;
+            agent.Logger.Finest($"AwsSdkPipelineWrapper: Request type is {requestType}");
 
-            agent.Logger.Finest("AwsSdkPipelineWrapper: Request type is " + requestType);
+            if (requestType.StartsWith("Amazon.SQS"))
+            {
+                return HandleSQSRequest(instrumentedMethodCall, agent, transaction, request, isAsync, executionContext);
+            }
+
+            agent.Logger.Debug($"AwsSdkPipelineWrapper: Unsupported request type: {requestType}. Returning NoOp delegate.");
```

Comment:
I'm not sure we need this and the above log line. This could be pretty verbose if they're using something besides SQS in the AWSSDK.

Reply:
You mean get rid of both lines or just one of them? I'm inclined to keep this one, though I could add some logic to only log the message once per type.

Reply:
Yeah, I think just one or the other would be fine.